data/systemd/snapd.service.in

```diff
@@ -17,6 +17,7 @@ OnFailure=snapd.failure.service
 OOMScoreAdjust=-900
 ExecStart=@libexecdir@/snapd/snapd
 EnvironmentFile=-@SNAPD_ENVIRONMENT_FILE@
+EnvironmentFile=-/var/lib/snapd/environment/snapd.conf
```

Comment:
Well, the idea is to eventually expand `snapd.conf` if that becomes necessary in the future. Meanwhile, this will be used only for SNAPD_DEBUG variable, so I didn't want to overcomplicate code by parsing it in a generic way.

Reply:
Just to be clear, the systemd parts looks ok.

Reply:
Fair enough, probably we need to align on this. Note though that we need to distinguish between options , which are directly controlled by snapd, so we would not allow others to touch them (`/var/lib/snapd/environment/snapd.conf` needs to be in sync with the value of options in snapd state) and those where we allow editting by others and we do not want them to be options.